build.sbt

```diff
@@ -55,17 +55,20 @@ ReleaseKeys.publishArtifactsAction := PgpKeys.publishSigned.value
 
 libraryDependencies ++= Seq(
   "com.metamx" %% "scala-util" % "1.11.3" exclude("log4j", "log4j") force(),
-  "com.ircclouds.irc" % "irc-api" % "1.0-0014"
+  "com.ircclouds.irc" % "irc-api" % "1.0-0014",
+  "org.apache.kafka" % "kafka-clients" % "0.9.0.0",
+  "ch.qos.logback" % "logback-core" % "1.1.2",
```

Comment:
TLDR: what you did is OK

Long version!

I originally excluded them because in Java logging, IMO the good-citizen way to behave is for apps to include logging (so people running the app can see stuff), but for libraries to log through a facade like slf4j and  *not* include a specific logging implementation. This is so people including the library can use their preferred logging framework (e.g. log4j) without conflicting with the one pulled in by the library (e.g. logback), and without having to exclude the library's logging framework manually.

There's a tough choice to make with packages like this that are both apps and libraries. I think the cleanest solution is to separate the packages (wikiticker-core and wikiticker-console) but since nobody really uses wikiticker other than us, that's overkill here and what you did is OK.

We do separate things like this for tranquility. Core (a library) does not include a logging impl, but Server/Kafka (apps) do depend on logback.